src/client/clientmap.cpp

```diff
@@ -1092,6 +1092,55 @@ void ClientMap::renderMap(video::IVideoDriver* driver, s32 pass)
 	g_profiler->avg(prefix + "material swaps [#]", material_swaps);
 }
 
+void ClientMap::invalidateMapBlockMesh(MapBlockMesh *mesh)
+{
+	ScopeProfiler sp(g_profiler, "CM::invalidateMapBlockMesh", SPT_ADD, PRECISION_MICRO);
+
+	// find all buffers
+	MeshBufListMaps tmp;
+	tmp.addFromBlock(v3s16(), mesh, getSceneManager()->getVideoDriver());
```

Comment:
Will this do the right thing with client_mesh_chunk > 1?

Reply:
I didn't test but it *should*